tests/rhui3_tests/tested_repos.yaml

```diff
@@ -0,0 +1,8 @@
+yum_repo1:
+    name: "Red Hat Update Infrastructure 2 \\(RPMs\\)"
+    version: " \\(6Server-x86_64\\)"
```

Comment:
IMHO, "version" should be just the version itself, without the space character at the beginning of the actual version string. The space character between the name and version should be inserted when (and only) the two variables are being put together. It means, delete the space character from the version, don't use:

yum_repo_name + yum_repo_version

and use this kind of concatenation instead:

yum_repo_name + " " + yum_repo_version

This is a much cleaner approach to data types. (Your last name doesn't begin with " ", either, and yet the space between your first name and last name appears when they're displayed together.)